pkg/controller/elasticsearch/driver/upgrade_forced_test.go

```diff
@@ -27,131 +27,131 @@ func Test_defaultDriver_maybeForceUpgradePods(t *testing.T) {
 		{
 			name: "no pods to upgrade",
 			actualPods: []corev1.Pod{
-				sset.TestPod{Name: "pod1", StatefulSetName: "ssetA", Ready: false}.Build(),
-				sset.TestPod{Name: "pod2", StatefulSetName: "ssetB", Ready: true}.Build(),
+				sset.TestPod{Name: "pod1", StatefulSetName: "ssetA", Ready: false, ResourceVersion: "999"}.Build(),
```

Comment:
I kept it configurable in case the future tests we add would actually need to modify the resource version. For the existing tests, yes, I agree that it can be hard-coded.